automatic/flashplayeractivex/tools/chocolateyInstall.ps1

```diff
@@ -1,14 +1,13 @@
-﻿$packageName = '{{PackageName}}'
-$version = '{{PackageVersion}}'
+$packageName = 'flashplayeractivex'
+$version = '24.0.0.186'
+$majorVersion = '24'
 $installArgs = '/quiet /norestart REMOVE_PREVIOUS=YES'
-$url = '{{DownloadUrl}}'
+$url = 'http://download.macromedia.com/get/flashplayer/current/licensing/win/install_flash_player_24_active_x.msi'
+$checksum = 'D1D60DF68B172F2CB21E4F8572BBB6727385BF4E5874DF95560947D35956BB9D'
+$checksumType = 'sha256'
 
-$majorVersion = ([version] $version).Major
-
-$alreadyInstalled = Get-WmiObject -Class Win32_Product | Where-Object {
-  $_.Name -eq "Adobe Flash Player $majorVersion ActiveX" -and
-  $_.Version -eq $version
-}
+$registry = ( Get-UninstallRegistryKey -SoftwareName Adobe Flash Player $majorVersion ActiveX ).DisplayVersion
+$alreadyInstalled = @{$true = "Adobe Flash Player ActiveX for IE $version is already installed."; $false = "Adobe Flash Player ActiveX for IE $version is not already installed."}[ $registry -ne $version ]
```

Comment:
this do the opposite of what you want.
this will think it is installed when it actually isn't.
`-ne = not equal`